server/views/case-summary.njk

```diff
@@ -20,7 +20,7 @@
 
 {% block javascripts %}
     {{ super() }}
-    <script type="text/javascript" src="/javascripts/case-progress-{{ appVersion }}.min.js"/>
```

Comment:
any reason this has been removed? You won't be able to take advantage of cache busting

Reply:
Good spot. I do this to put non minified version for debugging purpose and later forget to remove